### ISSUE TITLE
[Term Entry] SwiftUI View Modifier: `.background()`

### ISSUE THREAD

**Issue by yangc95** [Codecademy/docs#1510]
### Reason/inspiration (optional)

Please write a new entry under: content/swiftui/concepts/viewmodifier/terms/ about the modifier `.background()` in Swift.

A "Syntax" section with pseudo code snippet.
An "Example" section with a swift code snippet and image to accompany. Images should go in the media/ folder.

To get started with the PR and entry file:

Under content/swiftui/concepts/viewmodifier/terms/, create a new folder called `background`. Then create a new file called `background.md` under content/swiftui/concepts/viewmodifier/terms/`background`

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

**Comment by flodega** [comment]
Hello, could I work on this ?  I had requested to work on the color issue because it seemed the person assigned hadn't completed but I got no response.  Can I work on this if the other one is not available anymore ?  

I have already forked the repository, cloned it, created a new branch and just looking for an issue to work on now.

**Comment by Dusch4593** [comment]
Hey @flodega, just assigned this to you 😄 

Thank you for your interest in contributing to Codecademy Docs! 🚀

**Comment by flodega** [comment]
Hello, thank you for assigning this to me. I was assigned another issue as well. I will work on the other one first and when I'm done with that, I will work on this one if that's okay

**Comment by SSwiniarski** [comment]
@flodega, it's been two weeks. Are you still working on this issue? Do you need any assistance?

**Comment by flodega** [comment]
@SSwiniarski Yes, apologies for not replying to you earlier.  I originally requested this issue as an alternative to the "color" one.  But you assigned the color one to me and I am just waiting for a second review now.

**Comment by yangc95** [comment]
@flodega if you were concerned about having multiple PRs up, that's understandable. Did you still want to work on this issue or would you prefer your open PR to be processed first before taking on another issue?

**Comment by flodega** [comment]
@yangc95 I would prefer to have my open PR processed first before taking on another issue.  Thanks

**Comment by matthewogtong** [comment]
Can I be assigned this issue?

**Comment by SSwiniarski** [comment]
@matthewogtong you've been assigned!

**Comment by SSwiniarski** [comment]
@matthewogtong It's been two weeks, are you still working on this? Do you need any assistance?

**Comment by matthewogtong** [comment]
> 

@SSwiniarski Apologies for the delay, I've just submitted a PR for this issue. Please review and let me know if there are any adjustments you'd like me to make.
